include/osquery/dispatcher.h

```diff
@@ -67,52 +39,30 @@ class InterruptableRunnable {
 
  protected:
   /// Allow the runnable to check interruption.
-  bool interrupted();
+  virtual bool interrupted();
```

Comment:
I'd guess it should be const method, wouldn't it?

Reply:
Not really, in other classes, it's overridden and they are changing the state.(buffered_tests.cpp and dispatcher_tests.cpp)